### PR TITLE
Follow-up edits to PR #2292

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continues work from https://github.com/openshift/openshift-docs/pull/2482

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tpoitras @bfallonf Picking this up from Thien-Thi. PTAL. :bow:

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -1211,73 +1211,82 @@ endif::[]
...
-the router instance without rebuilding the router image. The
-*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be modified
-as well as creating and modifying router environment variables.
-
-The following example shows how to use a custom *_haproxy-config.template_* in
-your router.
+A xref:../../dev_guide/configmaps.adoc#dev-guide-configmaps[ConfigMap] permits
+you to customize the router instance without rebuilding the router image. The
+*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be
+modified as well as creating and modifying router environment variables.
```
Comment (bfallonf) [inline install_config/install/deploy_router.adoc]: @ahardin-rh I know this para was as-is when you got here, but I thin 'permit' has different connotations than what's intended here. Maybe "You can use ConfigMaps to customize the router instance without rebuilding the router image."

Also, oxford comma after "reload-haproxy".

File: install_config/install/deploy_router.adoc [hunk 2]
```diff
@@ -1211,73 +1211,82 @@ endif::[]
...
-The configMap contains a copy of your modified *_haproxy-config.template_* file.
-The router dc is next modified to mount the configMap as a file and the
-`TEMPLATE_FILE` environment variable is modified to point to it. This can be
-done using "oc env" and "oc volume" or alternaitvely by editing the router dc.
++
+The `customrouter` ConfigMap now contains a copy of the modified
+*_haproxy-config.template_* file.
 
-====
+. Modify the router deployment configuration to mount the configMap
```
Comment (bfallonf) [inline install_config/install/deploy_router.adoc]: Caps in "configmap"?

**Comment by bfallonf** [comment]
@ahardin-rh A couple comments, but otherwise LGTM

**Comment by ahardin-rh** [comment]
@bfallonf Thanks for the eyeballs! :eyes:

**Comment by ahardin-rh** [comment]
No rev history needed

**Pull request merged**